client/packages/common/src/intl/locales/en/common.json

```diff
@@ -327,7 +327,7 @@
   "error.plugin": "There was an error loading the plugin {{name}}",
   "error.plugin-unavailable": "Plugin unavailable",
   "error.prescription-not-found": "Prescription not found.",
-  "error.printing-qr": "Error printing QR code",
+  "error.printing-asset-label": "Error printing asset label code",
```

Comment:
Oo - could you make quick follow up PR @zachariah-at-msupply, think `code` needed to be removed here 🙏

Reply:
All done, the follow up PR is here...
https://github.com/msupply-foundation/open-msupply/pull/5853